tests/unit/test_exceptions.py

```diff
@@ -138,15 +138,15 @@ def test_from_http_response_bad_json_content():
 def test_from_http_response_json_unicode_content():
     response = make_response(
         json.dumps(
-            {"error": {"message": u"\u2019 message", "errors": ["1", "2"]}}
+            {"error": {"message": "\u2019 message", "errors": ["1", "2"]}}
         ).encode("utf-8")
     )
 
     exception = exceptions.from_http_response(response)
 
     assert isinstance(exception, exceptions.NotFound)
     assert exception.code == http.client.NOT_FOUND
-    assert exception.message == u"POST https://example.com/: \u2019 message"
```

Comment:
I found a few more of these in `docs/conf.py`, let's remove them there, too.

Reply:
Templated file -- I'd rather fix that upstream in `synthtool`.

Reply:
https://github.com/googleapis/synthtool/pull/1119